accessx-status/src/applet.c

```diff
@@ -1632,11 +1634,18 @@ accessx_status_applet_fill (MatePanelApplet* applet)
         was_realized = TRUE;
     }
 
-    g_object_connect (sapplet->applet,
-                      "destroy", accessx_status_applet_destroy, sapplet,
-                      "change-orient", accessx_status_applet_reorient, sapplet,
-                      "change-size", accessx_status_applet_resize, sapplet,
-                      NULL);
+
+    g_signal_connect (sapplet->applet, "destroy",
+                      G_CALLBACK (accessx_status_applet_destroy),
+                      sapplet);
+
+    g_signal_connect (sapplet->applet, "change-orient",
+                      G_CALLBACK (accessx_status_applet_reorient),
+                      sapplet);
+
+    g_signal_connect (sapplet->applet, "change-size",
+                      G_CALLBACK (accessx_status_applet_resize),
+                      sapplet);
```

Comment:
```suggestion
    g_object_connect (sapplet->applet,
                      "signal::destroy", accessx_status_applet_destroy, sapplet,
                      "signal::change-orient", accessx_status_applet_reorient, sapplet,
                      "signal::change-size", accessx_status_applet_resize, sapplet,
                      NULL);
```

Reply:
Either way if fine though, using the convenience only saves a bit of typing and duplication.

Reply:
Testing this now, I can see what you did different using` "signal::<something>"`

Reply:
Works fine both in-process and out, will commit this